pkg/kubelet/pleg/generic.go

```diff
@@ -309,7 +333,10 @@ func (g *GenericPLEG) Relist() {
 		for i := range events {
 			// Filter out events that are not reliable and no other components use yet.
 			if events[i].Type == ContainerChanged {
-				continue
```

Comment:
This prevents a `ContainerChanged`event from being sent when a container status is `created`, which is sometimes detected and converted to `unknown` at L99. Even if this event is sent, it doesn't seem to cause a problem at a glance. However, it would be better not to send an event when a container status is `created` in order to avoid any unexpected side-effects by using another event (`PodSync` or new one) for resizing or by checking the container status.